addons/l10n_be/models/template_be_ass.py

```diff
@@ -0,0 +1,27 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import _, models
+
+from odoo.addons.account.models.chart_template import template
+
+
+class AccountChartTemplate(models.AbstractModel):
+    _inherit = 'account.chart.template'
+
+    @template('be_ass')
```

Comment:
I think `be_ass` is a bit silly. It's ok for me to keep that "name" but do the others (mainly QDP and POs) agree with it?

Reply:
May I suggest be_asso ?

Reply:
It's part of the "funny messages" philosophy ... Come on! 😄 
No, for me it's fine changing it. So that could then be `be_asso` and `be_comp`?